cristin-commons/src/main/java/no/unit/nva/utils/LanguageMapperExtra.java

```diff
@@ -0,0 +1,16 @@
+package no.unit.nva.utils;
```

Comment:
we have a language library for such things. https://mvnrepository.com/artifact/com.github.bibsysdev/nva-language

Reply:
using nva-language..